### ISSUE TITLE
Make the repo oms-core with user vagrant

### ISSUE THREAD

**Issue by ingomueller-net** [AEGEE/oms-development-vm#22]
So with ssh one can go and pull the latest changes from git

alternatively I ask, what's the best way to update that repo in your opinion? the reason for which I had the line 
```
#!ruby

config.vm.synced_folder "oms-core", "/srv/oms-core"
```

in the vagrantfile at 7f5cbcc (even though commented) was to uncomment it after provisioning and have real time changes in the VM when I changed my code

----------------------------------------
- Bitbucket: https://bitbucket.org/ingomueller-net/ldap-poc/issue/22
- Originally reported by: [grasshopper](http://bitbucket.org/grasshopper)
- Originally created at: 2015-10-30T11:37:25.285

**Comment by ingomueller-net** [comment]
That's something that I don't see super clear yet. I'll investigate a bit what options we have... Since every developer has other preferences, we should probably offer several. Probably the synced folder is one, but then I would still deploy it with puppet...

----------------------------------------
Original comment by: Ingo Müller

**Comment by ingomueller-net** [comment]
Maybe an option: http://serverfault.com/questions/6895/whats-the-best-way-of-handling-permissions-for-apache2s-user-www-data-in-var

----------------------------------------
Original comment by: Ingo Müller

**Comment by ingomueller-net** [comment]
Maybe another one: http://www.virtadmin.com/vagrant-series-synced-folders-part-4/

----------------------------------------
Original comment by: Ingo Müller

**Comment by ingomueller-net** [comment]
Also interesting: http://askubuntu.com/questions/386928/default-permissions-for-var-www and http://serverfault.com/questions/124800/how-to-setup-linux-permissions-for-the-www-folder

----------------------------------------
Original comment by: Ingo Müller

**Comment by serviceman** [comment]
closed by ad5f431, I don't know why it wasn't automatically updated (only the first issue was)
